README.md

```diff
@@ -1,8 +1,27 @@
-Confluent's Apache Kafka client for Golang
-==========================================
+Confluent's Golang Client for Apache Kafka<sup>TM</sup>
```

Comment:
-1 on this change, I think the previous version reads better. (this is an Apache Kafka client for Golang, not a Golang client (?) for Apache Kafka).